templates/cleanup

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 ## Clean-up the keywordfile 
-sed -i "/@@CPV@@/d" @@KEYWORDFILE@@
+sed -i "/# Job @@CPV@@$/d" @@KEYWORDFILE@@
```

Comment:
Should this be ``# Job @@JOB@@$``?

Reply:
My testcase is "tatt -b 627718" on sparc. It ends up with multiple lines ending in "# Job libfastjson" in package.keywords, and the cleanup file just matches on "libfastjson" (which is just the first atom in the bug). So either the change is right or it has been wrong before ;)

Reply:
I should check the code before commenting next time, turns out we're both right :-)
```   
    script = cleanUpTemplate.read().replace("@@JOB@@", job.name)
    script = script.replace("@@CPV@@", job.name)
```